packages/styled-docs/pages/theme.mdx

```diff
@@ -1 +1,217 @@
+import { Box, Flex, Grid, Stack, useTheme } from "@trendmicro/react-styled-core";
+import {
+  ColorPalette,
+  ColorPalettes,
+  ColorWrapper,
+} from "../components/ColorPalette";
+import ThemeParser from "../components/ThemeParser";
+
 # Theme
+
+The theme object is where you define your application's color palette, type
+scale, font stacks, breakpoints, border radius values, and more. Theming is based on the
+[Styled System Theme Specification](https://styled-system.com/theme-specification)
+
+## Colors
+
+Add a `theme.colors` object to provide colors for your project. By default these
+colors can be referenced by the `color`, `borderColor`, and `backgroundColor`,
+`fill`, `stroke`, styles.
+
+We recommend adding palette that go from `10` - `100`.
+
+<ThemeParser theme="colors" />
+
+### Orange
+
+<ColorWrapper>
+  <ColorPalettes color="orange" />
+</ColorWrapper>
+
+### Yellow
+
+<ColorWrapper>
+  <ColorPalettes color="yellow" />
+</ColorWrapper>
+
+### Red
+
+<ColorWrapper>
+  <ColorPalettes color="red" />
+</ColorWrapper>
+
+### Magenta
+
+<ColorWrapper>
+  <ColorPalettes color="magenta" />
+</ColorWrapper>
+
+### Purple
+
+<ColorWrapper>
+  <ColorPalettes color="purple" />
+</ColorWrapper>
+
+### Blue
+
+<ColorWrapper>
+  <ColorPalettes color="blue" />
+</ColorWrapper>
+
+### Green
+
+<ColorWrapper>
+  <ColorPalettes color="green" />
+</ColorWrapper>
+
+### Teal
+
+<ColorWrapper>
+  <ColorPalettes color="teal" />
+</ColorWrapper>
+
+### Cyan
+
+<ColorWrapper>
+  <ColorPalettes color="cyan" />
+</ColorWrapper>
+
+### Gray
+
+<ColorWrapper>
+  <ColorPalettes color="gray" />
+</ColorWrapper>
+
+### BlackAlpha
```

Comment:
Can we change the section title to be meaningful rather than using BlackAlpha / WhiteAlpha?

Reply:
I think black transparent/white transparent is better.